Makefile

```diff
@@ -161,7 +161,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
 
 .PHONY: docker-build
 docker-build: ## Build docker image with the manager.
-	docker buildx build --build-arg VERSION=${VERSION} -t ${IMG} -t ${IMG_LATEST} . --load
+	docker buildx build --build-arg VERSION=${VERSION} -t ${IMG} -t k8ssandra/cass-operator:v${VERSION} -t ${IMG_LATEST} -t k8ssandra/cass-operator:latest . --load
```

Comment:
No hardcoding, can't override in scripts.

Reply:
I'm hardcoding the docker hub images. My first attempt had two sets of variables, one for Docker Hub and the other for GCP AR, it looked cluttered. I can move back to that model if that's what's preferred, but given we hardcoded the bits for the RH repos I thought this would be ok.

What's your preference?

Reply:
We don't use this command for publishing to any repository (the GHA has different builder properties). This is just for local building. For that reason, the IMG_LATEST should be fine (since its values can be overridden)